src/test/java/catalinazoldi/teorie/selenium2/JavaExceptions.java

```diff
@@ -0,0 +1,44 @@
+package catalinazoldi.teorie.selenium2;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+          public class JavaExceptions {
+              public static void main(String[] args) throws IOException {
+                  //checkedExceptions();
+                  //uncheckedException();
+                  //System.out.println("Am executat functia main ");
+              }
+
+              //checked exceptions
+              static void checkedExceptions() throws IOException {
+                  FileReader fileReader = new FileReader("C:\\Users\\cati\\IdeaProjects\\Procyon-QAC-G1-13.10.22\\src\\test\\java\\catalinazoldi\\teorie\\selenium2\\text.txt");
```

Comment:
Please don't use your absolute local path. The test won't run on other machines. Thanks! :)
This line should become -> FileReader fileReader = new FileReader((System.getProperty("user.dir") + "\\src\\test\\java\\catalinazoldi\\teorie\\selenium2\\text.txt");
In this way we are giving a relative path inside our project and it will work on every machine.